.github/workflows/main.yaml

```
@@ -126,21 +147,28 @@ jobs:
     - name: Build Python
       run: bash ./pythia build
 
-    - uses: actions/upload-artifact@v2
-      with:
-        name: pythia-macos
-        path: dist/**/*.tar.gz
-
     - name: Test Python
       run: bash ./pythia test
 
+    - name: Upload testing package
+      env:
+        BIN_PRIV_KEY: ${{ secrets.SFTPPLUS_BIN_PRIV_KEY }}
+        BIN_HOST_KEY: ${{ secrets.SFTPPLUS_BIN_HOST_KEY }}
+      run: |
+        mkdir -p ~/.ssh/
+        echo "$BIN_HOST_KEY" > ~/.ssh/known_hosts
```

I think that we can do this
```suggestion
        echo "${{ secrets.SFTPPLUS_BIN_PRIV_KEY }}" > ~/.ssh/known_hosts
```

---

Thanks, done.